.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
what's this red character?

Reply:
I think it's github, for some reason it complains because there's no newline. Adding one.